v3/src/components/slider/slider.scss

```
@@ -16,6 +17,10 @@ $slider-component-height: 98px;
   background: #fff;
   border-radius: vars.$border-radius-bottom-corners;
 
+  &.date {
```

```suggestion
  &.twoLevel {
```
Based on my suggestion above that the class name should reflect its meaning.